toolbox

```diff
@@ -69,16 +69,20 @@ run() {
         echo "Setting up user '${USER_NAME}' (with 'sudo' access) inside the container..."
         echo "(NOTE that, if 'sudo' and related packages are not present in the image already,"
         echo "this may take some time. But this will only happen now that the toolbox is being created)"
-        cat <<EOF > /tmp/${TOOLBOX_NAME}-user-setup.sh
+	local tmp_user_setup=`mktemp /tmp/${TOOLBOX_NAME}-user-setup-XXXXXX.sh`
+        cat <<EOF > ${tmp_user_setup}
 #!/bin/bash
 groupadd -g ${USER_GID} ${USER_GNAME} &> /dev/null
 useradd -M -N -g ${USER_GNAME} -u ${USER_ID} ${USER_NAME} &> /dev/null
-zypper install -y --no-recommends sudo system-group-wheel &> /dev/null
+if ! rpm -ql system-group-wheel &> /dev/null; then
```

Comment:
This does not need to work. The correct way to test for the wheel group is to use "getent group wheel >/dev/null || ..."

Reply:
Ah, yes, sure! Ok, I'll do it like this